CODEOWNERS

```diff
@@ -8,6 +8,6 @@ docs/guides/network_types.md @equinix/governor-metal-client-interfaces
 *connection_e2e*             @equinix/governor-digin-fabric
 *resource_network_*          @equinix/governor-ne-network-edge-engineering
 *data_source_network_*       @equinix/governor-ne-network-edge-engineering
-**/edge-networking           @equinix/governor-ne-network-edge-engineering
-docs/**/network_*            @equinix/governor-ne-network-edge-engineering
-templates/**/network_*       @equinix/governor-ne-network-edge-engineering
+edge-networking           @equinix/governor-ne-network-edge-engineering
+network_*                 @equinix/governor-ne-network-edge-engineering
```

Comment:
does this include metal_device_network_type?

Reply:
moved the override down so it'll actually work for once